locals.tf

```diff
@@ -65,5 +65,9 @@ locals {
   weekly_ci_disk_size    = 8
   weekly_ci_access_modes = ["ReadWriteOnce"]
 
+  release_ci_disk_size    = 64
```

Comment:
As checked together, you might want to update the PR here to apply the same pattern as in https://github.com/jenkins-infra/azure/pull/769 to avoid local and make the PVC -> PV -> Disk dependency chain explicit